src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -871,6 +875,26 @@ COMMENT ON COLUMN media_request.output_format
 IS 'format of the requested media object, possibly migrated from moj_transformation_s';
 
 
+CREATE TABLE node_register
+(node_id                     INTEGER                       NOT NULL  --pk column breaks pattern used, is not der_id

```

Comment:
This comment doesn't make sense anymore. It's also very confused, looking in dev local database - we need to tidy up the sequences. Why do we have sequence `der_seq` still?
Wondering should it be sequence `node_seq` because of the choice in PK column name.
We seem to be using `nod_seq` in the Java code currently, so the mismatch needs to be corrected still.

Reply:
all reference to device_register should now be removed, and node_register used consistently. the only deviation from the pattern now, is that the PK column is node_id rather than nod_id, as might be expected from the 3 chars used elsewhere. However, as node_id is a column present in legacy it was decided to retain the column name as is ( generally all legacy PKs are varchar, and are not being retained as PKs in the modernised tables, other than as an alternate key in places as means to verify migration)